modules/themes/src/main/java/org/opencastproject/themes/persistence/ThemesServiceDatabaseImpl.java

```diff
@@ -329,12 +329,7 @@ private ThemeDto getThemeDto(long id, EntityManager em) {
   }
 
   @Override
-  public void repopulate(final ElasticsearchIndex index) {
-    if (index.getIndexName() != this.index.getIndexName()) {
-      logger.info("Themes are currently not part of the {} index, no re-indexing necessary.");
-      return;
-    }
```

Comment:
Will this case not happen any more so that themes are always written to the index or is it handled elsewhere so that we don't need this option here?